addons/imagedecoder.heif/imagedecoder.heif.json

```diff
@@ -10,16 +10,8 @@
     ],
     "modules": [
         {
-            "name": "libheif",
+            "name": "libde265",
```

Comment:
Any reason, for switching the order of these two?

Reply:
yes, it;s described in [commit](https://github.com/flathub/tv.kodi.Kodi/pull/225/commits/012a9f137da8854e639060342d8a3919f07f4cb6):

> The older version doesn't build anymore.
> 
> Also build libde265 dependency before libheif as otherwise it's useless.